src/components/pages/Categories/List.vue

```diff
@@ -65,6 +77,32 @@ export default {
       if (this.loading) return;
       this.$store.dispatch('categories/postCategory');
     },
+    openModal(categoryId, categoryName) {
+      const deleteCategory = {
+        categoryId,
+        categoryName,
+      };
```

Comment:
`deleteCategory` というオブジェクト名の中に `categoryId` というプロパティ名があって、`category` という情報が重複してしまっていると思います。
以下のように記述するとシンプルになるかと思います。

```suggestion
      const deleteCategory = {
        Id: ・・・,
        name: ・・・,
      };
```

Reply:
[248fbd5](https://github.com/gizumo-education/gizumo_wiki_lesson/pull/175/commits/248fbd5028077d31b41f3c84d698797ee79ea7ed)